### DISCUSSION TITLE
Welcome to [cmd.ms] Discussions!

### DISCUSSION THREAD

**Discussion by merill** [merill/cmd#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by glecas** [comment]
Beautiful work! This will come in handy in my everyday work and will now be my top bookmark for working with Microsoft 365! I love this!

**Comment by merill** [comment]
Cheers @glecas super glad you like it.
